src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -693,6 +693,10 @@
     <Field Name="GetAwaiter" Type="BoundExpression?" Null="allow"/>
     <Field Name="IsCompleted" Type="PropertySymbol?" Null="allow"/>
     <Field Name="GetResult" Type="MethodSymbol?" Null="allow"/>
+    <!-- Refers to the runtime async helper we call for awaiting. Either this is an instance of an AsyncHelpers.Await call, and 
```

Comment:
>an instance of an AsyncHelpers.Await call [](http://example.com/codeflow?start=81&length=41)

1. we're not storing a call
2. consider also commenting on the null case

Reply:
3. Consider adding a `Validate` method with `HasValidate="true"`

Reply:
4. Let's take a note to revisit the `GetAwaitExpressionInfo` API which exposes various bits of `BoundAwaitableInfo`. 
5. It would also be good to review the various consumers of `BoundAwaitableInfo` and check if some can assert that `RuntimeAsyncAwaitMethod` is null (for example, `AsyncMethodToStateMachineRewriter.VisitAwaitExpression`)